package.json

```diff
@@ -30,7 +30,9 @@
     "grunt-contrib-watch": "~0.4.0",
     "grunt-umd": "^2.3.3",
     "load-grunt-tasks": "^3.1.0",
+    "lodash": "~3.10.1",
     "qunitjs": "^1.17.1",
-    "video.js": "^4.12.4"
+    "sinon": "~1.16.1",
```

Comment:
With sinon, I would even just get rid of the prefix altogether. They broke too many times on minor and patch releases to trust them.
lodash I think is fine to use `^`.